packages/docs/src/pages/startup/migrating-v4.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Migrating to v4
+weight: 11
+---
+
+CMS Design System v4 introduces several breaking changes, and this migration guide will outline several steps to upgrade to the latest version.
+
+## NPM packages
+
+The `@cmsgov/design-system-core`, `@cmsgov/design-system-support`, `@cmsgov/design-system-layout` npm packages have been deprecated and replaced with a new consolidated package [`@cmsgov/design-system`](https://www.npmjs.com/package/@cmsgov/design-system). This is now the only dependency you need to use the design system and you can simply replace the old packages with `@cmsgov/design-system` in your `package.json`.
+
+The new file structure below illustrates where the old `support` and `layout` packages can now be found.
+
+```
+cmsds@4.0.0
```

Comment:
Should this include `dist` at the top level?

Reply:
I think it depends on when we are deciding to update the doc site with this migration guide. If we are waiting until after the scripts are done, i dont think we need the `dist` because we arent including it in the release anymore right?

Reply:
Oh I realized the documentation on this page references `dist` a lot, I think I misunderstood. I thought that once we move the SCSS src and assets inside the `dist` folder, we can just release the `dist` folder as the package?

Reply:
This would have the other added benefit of users not having to update their fonts and image path variables, because although its changed on our end (moved to `dist`), if we just distribute `dist` then its the same for the users

Reply:
Update: We had a slack discussion about this and concluded that the package would still need to have a `dist` folder in it but that it just wouldn't include a `src` folder anymore.